### ISSUE TITLE
publish to s3

### ISSUE THREAD

**Issue by trvsdnn** [trvsdnn/frank#41]

**Comment by phrozen** [comment]
Nanoc supports this via fog, and not only Amazon S3, but Google Storage and Rack Space too thanks to fog compatibility with those services. Maybe look into fog to provide this functionality. This will be an awesome feature to have.

https://github.com/fog/fog

**Comment by trvsdnn** [comment]
Yeah, I've used fog for a few other projects. It could be a good fit here. I'm thinking I might have some time to work on frank 2 sometime soon... been collecting ideas fo a while now.
